pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/BrokerResponse.java

```diff
@@ -24,11 +24,12 @@
 /**
  * Reimplementation of BrokerResponse from pinot-common, so that pinot-api does not depend on pinot-common.
  */
-class BrokerResponse {
+public class BrokerResponse {
```

Comment:
@KKcorps @npawar I would call this as a major change in the public API for java client. BrokerResponse was package protected but now this PR makes the class public. Do you know why the BrokerResponse was not public?

My understanding is the client was exposing the results, much like the relational databases so that the client application doesn't need to understand the internals of Pinot response but this requirement kind of beats that. The intention here is "I understand response stats and let me access them". 
Thoughts, please?

Reply:
I think thee intention was to expose a standard ResultSet Interface from which the user can easily read the data.
Broker Response can have different fields depending upon whether SQL or PQL endpoint was queried.

Reply:
IMO, ResultSet makes sense for JDBC client but for the Java client, the entire BrokerResponse makes more sense.

Reply:
Fixed this now by exposing stats in the ResultSetGroup itself. Please check.